deployment/ib-kubernetes-ufm-secret.yaml

```diff
@@ -1,12 +1,14 @@
 apiVersion: v1
+data:
+  UFM_ADDRESS: ci1kY3M1LTAy # 127.0.0.1 - base64 format
+  UFM_CERTIFICATE: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUQ2ekNDQXRPZ0F3SUJBZ0lKQU56eVQxUlN4amkrTUEwR0NTcUdTSWIzRFFFQkN3VUFNSUdMTVFzd0NRWUQKVlFRR0V3SkpUREVMTUFrR0ExVUVDQXdDU1V3eER6QU5CZ05WQkFjTUJsSkJUa0ZPUVRFUk1BOEdBMVVFQ2d3SQpUV1ZzYkdGdWIzZ3hEREFLQmdOVkJBc01BMDFVVWpFYU1CZ0dBMVVFQXd3UlRtRnRaU0EyWVROaU5XRmlORFZqClltUXhJVEFmQmdrcWhraUc5dzBCQ1FFV0VrQk9ZVzFsSURaaE0ySTFZV0kwTldOaVpEQWVGdzB4T1RBNU1qVXcKT1RBM01EZGFGdzB5T1RBNU1qSXdPVEEzTURkYU1JR0xNUXN3Q1FZRFZRUUdFd0pKVERFTE1Ba0dBMVVFQ0F3QwpTVXd4RHpBTkJnTlZCQWNNQmxKQlRrRk9RVEVSTUE4R0ExVUVDZ3dJVFdWc2JHRnViM2d4RERBS0JnTlZCQXNNCkEwMVVVakVhTUJnR0ExVUVBd3dSVG1GdFpTQTJZVE5pTldGaU5EVmpZbVF4SVRBZkJna3Foa2lHOXcwQkNRRVcKRWtCT1lXMWxJRFpoTTJJMVlXSTBOV05pWkRDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQwpnZ0VCQUxncDcwcExJcVlnaUxtNW5WRkk0cEdaRDFKcEYyU08vM0FmUlhMYVdQbHZyMUNQSGg2ajVYVjJMU0pyCjRSK25MRjZTTWlidUIvclcwMWVyMGJvaTNFZ3pLZy8yVEtoU1hjSk1WMUpwNGJHRlg0M3YyZGp1NFJXOVVTdFgKRzRPb0pyMUxObkV3TEl5azRoTkhRZzlVdWYvcS9xMStrblE5U09yMExadFN5MDBYeTNHalZIajM4em8veGVqZwpRQW1MbUJpeDFMME94OVRPY3dXcm1WdEQ3QUhWNmlhVkFaMTNPODkwY3ZCNzZkUXNpYmNYTlloeGJzeFNVOUtZCk9BTzhJRDVLSUQxS1V3bnd6Z2hxa29leVQ3Z1hrUFJrbnhHTzVjQ2oyU05jS25jczZvL0JkVGQxMEdlbFFBamwKUTBkSUoyRzk0YVJMendSVjZSUnBzSmx4WUZFQ0F3RUFBYU5RTUU0d0hRWURWUjBPQkJZRUZORXkzK3lUWUdBYgo3MUFYR1U2Z3hQK1pMRmFITUI4R0ExVWRJd1FZTUJhQUZORXkzK3lUWUdBYjcxQVhHVTZneFArWkxGYUhNQXdHCkExVWRFd1FGTUFNQkFmOHdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBQlpxd3RsajBtR3ZKdURublpHelBhT3UKRjVIdWdKL1lVQXM4QnlWanI4NjZMOE9KUWM4czZnbHh3WnR1Y01yaWxkR0JjT0tLeTJjbjJmRHh3T2ZjMnJRdwp3MmU4bVdidllEd0prYStkRmNsWm1EL1l2NU5XNWVIeFc3a3NwWXBjZ0ZZYlFWQVlxeHB1M0t0ZEJtSlduYlRHCkdqak1yZlFXeHZKOENwSi9wUWZZY3NxelVKZ3RBMGMwL0JVN1NlaVplc1pjK2g3RE1yUjYzZUNhYjgveHJWTG8KcVBUQmdCTzVQRmlvNCt0WVgxUUZ0dmRvTE1pRVpwRlRjbG5LNjl2Mlg1Nk90czRXSldLdlRvSWZFWkgzRkhyRQpISHRVT2NoUFN4MUtSTThmS1VUUHVLSUh0VWlQZGlSWkpROERpRHJ6a2k5VWdLazRGRjVIRTAvaG9BWjk1OEE9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K # /etc/pki/tls/certs/ufmlocalhost.crt - base64 format 
```

Comment:
why is this here?

Reply:
This is the base64 of the default certificate.  Now in https you have to provide one

Reply:
1. you need to cat <cert> | base64 > CERTIFICATE
and use the kubectl create secret generic ib-kubernetes-ufm-secret --from-file=UFM_CERTIFICATE. 
2. we use base64 to avoid special characters user can use none base64 format if he put it under stringData. 
see https://kubernetes.io/docs/concepts/configuration/secret/

Reply:
README should be updated as well on the process, but this can be another PR

Reply:
Let review the README in another PR